NEWS.md

```diff
@@ -1,3 +1,7 @@
+## 22.0.7 2021-12-21
+
+* Upgrade Log4j to v2.16.0 (CIRC-1387)
```

Comment:
https://issues.folio.org/browse/CIRC-1387 is for Kiwi upgrading from 22.1.3 to 22.1.4.
https://issues.folio.org/browse/CIRC-1390 is for Juniper upgrading from 22.0.6 to 22.0.7.

Reply:
done